src/IFX007T-Motor-Control.cpp

```diff
@@ -165,27 +231,38 @@ void IFX007TMotorControl::configureBLDCMotor(BLDCParameter MyParameters)
     _debugPin = 1;
     
 
-    if(MyParameters.SensingMode)    //Hall-sensor mode (Not supported yet)
+    if(MyParameters.SensingMode)    //Hall-sensor mode
     {
         #define HALLmode
-        pinMode(_PinAssignment[AdcPin][0], INPUT_PULLUP);
-        pinMode(_PinAssignment[AdcPin][1], INPUT_PULLUP);
-        pinMode(_PinAssignment[AdcPin][2], INPUT_PULLUP);
+        DEBUG_PRINT_LN("Hallsensor mode");
+        _CurrentDutyCycle = 80;     // dutycycle at the beginning
+
+        //pinMode(_PinAssignment[AdcPin][0], INPUT_PULLUP);
+        //pinMode(_PinAssignment[AdcPin][1], INPUT_PULLUP);
+        //pinMode(_PinAssignment[AdcPin][2], INPUT_PULLUP);
+
+        // Enable interrupts for hallsensor inputs
+        //PCICR  = 0b00000010;               // Enable pin change interrupt for pins 14:8 (See Atmel 328 datasheet, page 79)
+        //PCMSK1 = 0b00001110;               // Enable pin change interrupt for pins A1 (PCINT9), A2 (PCINT10), A3 (PCINT11)
```

Comment:
Wieso ist das alles auskommentiert?
Wenn das nicht gebraucht wird schmeiß es raus.

Reply:
You're right, it's a leftover of trying to implement an interrupt mode. Maybe relevant for the next release, but for now I deleted it.